lib/tasks/test-sass-compilation.js

```diff
@@ -5,42 +5,72 @@ const buildSass = require('./build-sass');
 const Listr = require('listr');
 const ListrRenderer = require('../helpers/listr-renderer');
 const isCI = require('is-ci');
+const { camelCase } = require('lodash');
+const ftSass = require('@financial-times/sass');
+const fs = require('fs');
+const execa = require('execa');
 
 async function compilationTest(cwd, { silent, brand } = {
 	silent: false,
 	brand: false
 }) {
-	const silentSass = require('../plugins/silent-sass');
 	const mainSassFilePath = await files.getMainSassPath(cwd);
 	if (!mainSassFilePath) {
 		return;
 	}
-	const sassFiles = await files.getSassFilesList(cwd);
-	const supportsSilent = await files.sassSupportsSilent(sassFiles, cwd);
-	if (!supportsSilent) {
-		return;
-	}
 
 	const name = await files.getComponentName();
-	const sassVar = '$' + name + '-is-silent: ' + String(Boolean(silent)) + ';';
-
-	const sassConfig = {
-		sassPrefix: sassVar,
-		sassIncludePaths: [cwd],
-		brand: brand,
-		buildFolder: 'disabled',
-		cwd: cwd,
-		sourcemaps: false,
-		outputStyle: 'compressed' // to remove sass multiline/loud comments from the css output
-	};
+	const primaryMixinName = camelCase(name);
+	const primaryMixinErrorCode = `OBT_NO_PRIMARY-MIXIN-${primaryMixinName}`;
+	const sassContent = fs.readFileSync(mainSassFilePath, 'utf8');
```

Comment:
`obt test` runs all the test sub-tasks at the same time, having this use a synchronous file read will cause the other sub-tasks to not run whilst this file is being read. Can we make this use an async version instead, that way other tasks/sub-tasks can run whilst the sass is being tested.

We would need to require the promisified version of `fs` - `require('fs').promise` and then use it like `await fs.readFile`